ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py

```diff
@@ -0,0 +1,30 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# as_dtype
+@handle_frontend_test(
+    fn_tree="torch.clone",
+    input_dtype=helpers.get_dtypes("valid", full=False),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_as_dtype(
```

Comment:
This should be renamed to `test_torch_clone`

Reply:
Noted!